src/api2/api-caller.js

```diff
@@ -211,13 +227,17 @@ export class ApiCaller {
       throw middlewares.parseJsonapiError(e)
     }
 
-    response = middlewares.parseJsonapiResponse(response)
-
-    if (!isEmpty(response.links)) {
-      if (opts.needSign) {
-        response.makeLinkCallersWithSignature(this)
-      } else {
-        response.makeLinkCallers(this)
+    if (opts.needRawResponse) {
```

Comment:
`false` is default, let’s invert the condition and bodies

Reply:
Inverted